snippets/ai_content_service/README.md

```diff
@@ -0,0 +1,121 @@
+# Building a Secure AI Content Service
+
+With multiple teams building GenAI applications, we are seeing several organizations adopt a central content/data model, where a single repository of information serves as the knowledge base for all internal AI applications across the company.
+
+While this centralized approach creates a single source of truth for all applications, it also increases the risk of data exposure if any of these applications get compromised. One way to mitigate this risk is to enforce the principle of least privilege (PoLP), giving applications only the minimum permissions required.
+
+Assuming you are using MongoDB as your central data store, let's explore how you can adopt PoLP in MongoDB using the document model itself.
```

Comment:
"MongoDB as a central data store"

This the first time I am seeing the use of central